src/Libm.jl

```diff
@@ -8,6 +8,9 @@ export xatan2, xasin, xacos, xatan, xsin, xcos, xsincos, xtan, xpow, xsinh, xcos
 # higher accuracy functions
 export xatan2_u1, xasin_u1, xacos_u1, xatan_u1, xsin_u1, xcos_u1, xsincos_u1, xtan_u1, xcbrt_u1, xlog_u1
 
+# alias for supported floating point types
+typealias FTypes Union{Float32,Float64}
```

Comment:
maybe it's better to leave as is since this package doesn't deal with Blas and for clarity.